### PR TITLE
Add Bors, dependabot and funding info

### PR DESCRIPTION
**Pull request by Yatekii**

### CODE REVIEW

File: .github/bors.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+required_approvals = 1
+block_labels = ["wip"]
+delete_merged_branches = true
+status = ["ci-linux (stable)", "ci-linux (nightly, true)"]
```
Comment (adamgreig) [inline .github/bors.toml:4]: do we want to block bors merging on nightly passing?

Comment (Yatekii) [reply .github/bors.toml:4]: Oh, no I guess we should remove that requirement :)

Comment (Yatekii) [reply .github/bors.toml]: Fixed!

**Review by adamgreig — APPROVED** [review]
bors r+

**Pull request merged**